packages/api-reference/src/features/Search/SearchModal.vue

```diff
@@ -108,7 +119,14 @@ function getFullUrlFromHash(href: string) {
 <template>
   <ScalarModal
     :state="modalState"
-    variant="search">
+    variant="search"
+    @keydown.down.stop="navigateSearchResults('down')"
```

Comment:
We probably want `.prevent` on these too so we don't move the cursor when going through results

Reply:
done